packages/fluentui/react-icons-northstar/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "../../../scripts/typescript/tsconfig.fluentui",
+  "compilerOptions": {
+    "composite": true,
+    "outDir": "dist/dts",
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noImplicitAny": true,
+    "noUnusedParameters": true,
+    "strictNullChecks": true
+  },
+  "include": ["src", "test"],
+  "references": [
+    {
+      "icons": "../icons"
```

Comment:
This is wrong 💣 

It should contains only `paths` to referenced local packages, https://www.typescriptlang.org/docs/handbook/project-references.html#what-is-a-project-reference

If you will keep there `createSvgIcon`:

```json
  "references": [
    {
      "path": "../accessibility"
    },
    {
      "path": "../react-bindings"
    },
    {
      "path": "../styles"
    },
  ]
```

Please also add `@fluentui/accessibility` in this case to dependencies:

![image](https://user-images.githubusercontent.com/14183168/77912190-65630000-7292-11ea-92f7-9e440a5c58f3.png)